### ISSUE TITLE
[BUG] Error loading ghostfolio (selfhosted) - since helmet middleware release

### ISSUE THREAD

**Issue by LeonStoldt** [ghostfolio/ghostfolio#2102]
**Bug Description**
Hi @dtslvr ,
I just updated to version [1.283.0](https://github.com/ghostfolio/ghostfolio/releases/tag/1.283.0) and noticed a whitescreen with errors happening while loading ghostfolio. (errors listed below)

I looked into the release notes and thought that the added helmet middleware looks quite promising for this cause. If I would setup ghostfolio behind my reverse proxy and serve it via https it would work - I guess. Currently, I am serving it only via http in my local network.
Is there a way to configure http as allowed? I guess some other users who don't have a reverse proxy present have the same problem as well and could not solve it that easily.

**To Reproduce**
1. Host ghostfolio in the local network with version <=[1.282.0](https://github.com/ghostfolio/ghostfolio/releases/tag/1.282.0) - last working version (I verified it by deploying version 1.282.0 again)
2. access it via http
3. update to version >=  [1.283.0](https://github.com/ghostfolio/ghostfolio/releases/tag/1.283.0)
4. access it via http and notice errors happening

**Expected behavior**

We should find some solution to either configure the middleware to allow http in local networks or disable it for local networks?
I haven't had time to look into the helmet middleware and the changes for this version switch in depth, but I would guess you have some sort of spontaneous idea on how to solve it or how the new middleware behaves.

**Screenshots**
<img width="653" alt="image" src="https://github.com/ghostfolio/ghostfolio/assets/31759512/0e159bb6-df6d-42c8-a8dc-bf820ad092a8">

**Logs**
Following errors occur in javascript console (see screenshot):
```
The Cross-Origin-Opener-Policy header has been ignored, because the URL's origin was untrustworthy. It was defined either in the final response or a redirect. Please deliver the response using the HTTPS protocol. You can also use the 'localhost' origin instead. See https://www.w3.org/TR/powerful-features/#potentially-trustworthy-origin and https://html.spec.whatwg.org/#the-cross-origin-opener-policy-header. - activities:1

The page requested an origin-keyed agent cluster using the Origin-Agent-Cluster header, but could not be origin-keyed since the origin 'http://myServer.local:3333' had previously been placed in a site-keyed agent cluster. Update your headers to uniformly request origin-keying for all pages on the origin. - activities:31

GET https://myServer.local:3333/ionicons/ionicons.esm.js net::ERR_SSL_PROTOCOL_ERROR
[...]
```

The docker container logs seem to be quiet.

**Environment**
- Self-hosted (Unraid) - normal linux docker env
- Ghostfolio Version 1.283.0
- any client

**Comment by dtslvr** [comment]
Thank you for the report @LeonStoldt. I have just released Ghostfolio [1.283.5](https://github.com/ghostfolio/ghostfolio/releases/tag/1.283.5) without the helmet middleware enabled by default.

**Comment by LeonStoldt** [comment]
I can confirm, that it works with 1.283.5.
Thank you for the quick fix!
I'll close this issue.
